cmd/vicadmin/server.go

```diff
@@ -439,13 +439,14 @@ func (s *server) bundleContainerLogs(res http.ResponseWriter, req *http.Request,
 	c, err := s.getSessionFromRequest(context.Background(), req)
 	if err != nil {
 		log.Errorf("Failed to get vSphere session while bundling container logs due to error: %s", err.Error())
+		if strings.Contains(err.Error(), "Could not validate session") {
```

Comment:
@vburenin if that's coming from vSphere, then yes. If it's from govmomi I don't think so.
@gigawhitlocks What's the real impact of redirecting to logout (and handling the case where you're not actually logged in) for ALL errors? I'm not seeing the alternative path added much value at this time.